py/LICENSE

```diff
@@ -0,0 +1,14 @@
+This directory is licensed differently from other parts of pathfinder due to
+`cairo-lang` license.
+
+The license is effectively unknown at this point. Code commited by Equilibrium
```

Comment:
```suggestion
The license is effectively custom at version 0.10. Code commited by Equilibrium
```

Reply:
Removed the whole license file because it was not what the license files are meant for.